mix.exs

```diff
@@ -19,6 +20,18 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [contributors: ["José Valim", "Milton Mazzarri", "Friedel Ziegelmayer",
+                   "eksperimental", "John Warwick", "Rafael Mendonça França",
+                   "Alexei Sholik", "Eric Meadows-Jönsson", "Peter Minten", "Sergey Nartimov",
+                   "Samuel Tonini", "Joseph Wilk", "Dave Thomas", "Tony Han",
+                   "Josh Adams", "Andrew Forward", "Brian Gianforcaro", "Kevin Montuori",
+                   "Lau Taarnskov", "Pedro Medeiros", "rightfold", "Stefan", "Vicent Marti",
+                   "Yuki Ito", "Chris Hyndman", "Thomas", "lidashuang"],
```

Comment:
Sorry, maybe I was not clear. :( I would leave this empty OR keep the current team.